### PR TITLE
Clean up formats in marketing topics folder (part 2)

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This was primarily as a fix for #22 (convert tables in marketing folder), but there are other formatting and style issues to address in these topics.

- Converted remaining HTML tables to MD
- Replaced H2/H3 infinitives with present tense verb headings (better for machine translation and findability)
- Adjusted some procedures to better use numbered steps, as well as clarify navigation and simplify the instructions
- Added alt text for images where needed
- A few misc. formats to align with MD linting rules

_Note:_ To keep the PR from being too large, this is the second of a series to address #22

**_Also:_** This is not a comprehensive rewrite! Other style edits can be done at a later time.

## Affected documentation pages

https://docs.magento.com/m2/ee/user_guide/marketing/gift-registries.html
https://docs.magento.com/m2/ee/user_guide/marketing/gift-registry-configure.html
https://docs.magento.com/m2/ee/user_guide/marketing/gift-registry-create.html
https://docs.magento.com/m2/ee/user_guide/marketing/gift-registry-information.html
https://docs.magento.com/m2/ee/user_guide/marketing/gift-registry-search.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-adwords.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-content-experiments.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-enhanced-ecommerce.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tag-manager-create-tag.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tag-manager-troubleshooting.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tag-manager.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-universal-analytics.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/invitations-configure.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/invitations.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/magento-product-reviews-config.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/magento-product-reviews-moderate.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/magento-product-reviews.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/marketing-menu.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/markup-tags-links.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/markup-tags.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/merchandising.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/meta-canonical.html
https://docs.magento.com/m2/ee/user_guide/marketing/google-tools.html
https://docs.magento.com/m2/ee/user_guide/marketing/meta-data.html
https://docs.magento.com/m2/ee/user_guide/marketing/newsletter-configuration.html
https://docs.magento.com/m2/ee/user_guide/marketing/newsletter-queue.html
https://docs.magento.com/m2/ee/user_guide/marketing/newsletter-subscribers.html
https://docs.magento.com/m2/ee/user_guide/marketing/newsletter-template.html
https://docs.magento.com/m2/ee/user_guide/marketing/newsletters.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rule-cart-scheduled-changes.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rule-catalog-scheduled-changes.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rule-discount-first-purchase.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rule-discount-minimum-purchase.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rule-multiple-sku.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-cart-buy-x-get-y-free.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-cart-coupon-code-configure.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-cart-coupon-report.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-cart-coupon.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-cart-create.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-cart-free-shipping.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-cart.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-catalog-create.html
https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-catalog.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Links to Magento source code or PRs

https://github.com/magento/merchdocs/pull/102

## Additional information

Internal staging build 271

### CODE REVIEW

File: src/marketing/google-tag-manager-create-tag.md [hunk 1]
```diff
@@ -7,65 +7,71 @@ If you have a Google AdWords account, you can create a tag that tracks conversio
...
+    ![Google Tag Manager - fire on]({% link images/images-ee/google-tag-manager-create-adwords-tag4.png %}){: .zoom}
+    _Fire On Triggers_
 
 1. In the Choose Pages section, complete the following settings:
 
     |Name:|Enter a name for the page description.|
     |Variable:|url|
-    |Operation:|matches RegEx. To learn more, see: What is “matches regex” for? in Tag Manager Help|
+    |Operation:|matches RegEx. (To learn more, see [What is “matches regex” for?][3] in Tag Manager Help.)|
     |Value:|checkout/success.*|
```
Comment (hickskenh) [inline src/marketing/google-tag-manager-create-tag.md]: i don't see a * explanation

File: src/marketing/google-tag-manager.md [hunk 2]
```diff
@@ -20,36 +20,36 @@ See Google's [Set up Site Search][7] for the basics you need to get started. Als
 
 1. To enable **Internal Site Search Tracking**, do the following:
 
-    * Navigate to **Select View** > **View Settings**.
+    - Navigate to **Select View** > **View Settings**.
 
-    * Turn **Site Search Tracking** On.
+    - Turn **Site Search Tracking** On.
```
Comment (hickskenh) [inline src/marketing/google-tag-manager.md]:
```suggestion
    - Set **Site Search Tracking** to `On`.
```

File: src/marketing/price-rules-catalog-create.md [hunk 3]
```diff
@@ -55,133 +55,135 @@ Follow these instructions to apply a discount to specific products whenever a se
...
 
-   |||
-   |--- |--- |
    |Apply as percentage of original|Discounts item by subtracting a percentage of the regular* price. For example: Enter 10 in Discount Amount for an final** price that is marked down 10% from the regular price.|
    |Apply as fixed amount|Discounts item by subtracting a fixed amount from the regular price. For example: Enter 10 in Discount Amount for a final price that is $10 less than the regular price.|
    |Adjust final price to this percentage|Adjusts the final price by a percentage of the regular price. For example: Enter 50 in Discount Amount for a final price that is marked down 50% from the regular price.|
-   |Adjust final price to discount value|Sets the final price to a fixed, discounted amount. For example:Enter 20 in Discount Amount for a final  price of $20.00.|
+   |Adjust final price to discount value|Sets the final price to a fixed, discounted amount. For example: Enter 20 in Discount Amount for a final price of $20.00.|
    ||Regular price refers to the base product price without any advanced pricing  (special/tier/group) or promotional discounts. Final price refers to the discounted price that appears in the shopping cart.|
```
Comment (hickskenh) [inline src/marketing/price-rules-catalog-create.md]: this row doesn't have a field name

File: src/marketing/gift-registry-search.md [hunk 4]
```diff
@@ -3,87 +3,87 @@ conditions: Default.EE-B2B
...
 
-1. When prompted to refresh the page cache, tap the link in the message at the top of the workspace and follow the instructions.
+    ![Gift registry - widget options]({% link images/images-ee/widget-gift-registry-search-widget-options.png %}){: .zoom}
+    _Widget Options_
+
+1. When complete, click <span class="btn">Save</span>.
+
+1. When prompted to refresh the page cache, click the link in the message at the top of the workspace and follow the instructions.
 
 ## Field Descriptions
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/marketing/gift-registry-search.md]: I am going to keep this one as separated tables for sections. I'm not really happy with the way we have the subsections on rows and it ends up looking really jumbled. Will probably have to go back to the configuration pages and do more of the same to make it more readable.

File: src/marketing/google-tag-manager-create-tag.md (hunk 1 above)
Comment (leslietilling) [reply src/marketing/google-tag-manager-create-tag.md]: I think that is a "wildcard" in the RegEx. That is part of their syntax. Think we should probably use code for these.

File: src/marketing/price-rules-catalog-create.md (hunk 3 above)
Comment (leslietilling) [reply src/marketing/price-rules-catalog-create.md]: I think those are from the * and ** items. It only refers to items in the one cell, so I think I will move this there. I hate footnotes, so happy to be rid of that anyway.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**